flash_examples/finetuning/tabular_classification.py

```diff
@@ -15,10 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from pytorch_lightning.metrics.classification import Accuracy, Precision, Recall
+
+import flash
+from flash.core.data import download_data
 from flash.tabular import TabularClassifier, TabularData
 
 # 1. Download the data
-download_data("https://pl-flash-data.s3.amazonaws.com/titanic.zip", 'data/')
+download_data("https://pl-flash-data.s3.amazonaws.com/titanic.zip", "data/")
```

Comment:
lets cache the data folder

Reply:
It should be caching `data/` directory already as I fixed it in https://github.com/PyTorchLightning/lightning-flash/pull/107.